screens/Map.js

```diff
@@ -82,35 +91,56 @@ export default class Map extends React.Component {
     });
   };
 
-  handlePress = marker => {
+  setRegionAndSelectLandmark = (event, marker) => {
+    this.setState({
+      region: {
+        ...event.nativeEvent.coordinate,
+        latitudeDelta: 0.005,
+        longitudeDelta: 0.005
+      }
+    });
+
     this.props.selectLandmark(marker);
   };
 
   render() {
     return (
-      <>
-      <MapView
-        provider={PROVIDER_GOOGLE}
-        style={specificStyles.mapContainer}
-        showsUserLocation={
-          this.state.locationResult === "denied" ? false : true
-        }
-        showsMyLocationButton={true}
-        zoomEnabled={true}
-        initialRegion={this.state.initialRegion}
-      >
-        {this.props.markers.map(marker => (
-          <MapView.Marker
-            key={marker.name}
-            coordinate={marker.coordinate}
-            title={marker.name}
-            onPress={() => this.handlePress(marker)}
-          />
-        ))}
-      </MapView>
-       {/* @TODO: refactor menu so list screen doesn't unmount the map and therefore reset initialRegion */}
-       <MenuBtn setScreen={this.props.setScreen} />
-       </>
-    )
+      <View>
```

Comment:
Wrapped everything in a View tag. We had some weird empty tags hanging out here before.